src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -664,7 +664,7 @@ private bool BuildFromNuspec(string path)
             if (_packArgs.InstallPackageToOutputPath)
             {
                 string outputPath = GetOutputPath(packageBuilder, _packArgs);
-                successful = BuildPackage(packageBuilder, outputPath: outputPath, symbolsPackage: false);
```

Comment:
Same here, no functionality changes, and I'm not sure it helps readability too much.